src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -108,7 +108,19 @@ public static AEItemStack fromPacket(final PacketBuffer buffer) {
         final boolean isCraftable = buffer.readBoolean();
         final long stackSize = buffer.readVarLong();
         final long countRequestable = buffer.readVarLong();
-        final ItemStack itemstack = buffer.readItemStack();
+
+        // based on buffer.readItemStack()
+        // Adapted to also handle forge capabilities
+        ItemStack itemstack = ItemStack.EMPTY;
+        if (buffer.readBoolean()) {
+            int i = buffer.readVarInt();
+            int j = buffer.readByte();
+            CompoundNBT nbt = buffer.readCompoundTag();
+            CompoundNBT forgeCap = buffer.readCompoundTag();
+
+            itemstack = new ItemStack(Item.getItemById(i), j, forgeCap);
+            itemstack.readShareTag(nbt);
```

Comment:
We're serializing the full tag, but read the share-tag here... Why?

Reply:
No idea. That's how vanilla/forge works and I don't like to change it and caus a different bug.

Reply:
We should be aware that we're breaking their contract here.
They allow items to override readShareTag, and they might expect that they do get a share tag, and not the full server tag.....